scripts/npm/prepublish/index.js

```diff
@@ -219,6 +221,12 @@ function publishComponents(result) {
   return Promise.all(components.map(util.publish));
 }
 
+function updateDocs(result) {
+  if (result && result.stdout) util.printLn.console(result.stdout);
+  if (!compontentsToPublish.length) return;
+  return util.docs(util.pkg.version);
```

Comment:
Be sure and add a `docs: require('./docs')` line in `lib/index.js` in order for `util.docs()` to work.